sampleconfig/configtx.yaml

```diff
@@ -72,6 +72,31 @@ Organizations:
             - Host: 127.0.0.1
               Port: 7051
 
+        # PrivateDataImplicitCollection stores the configuration for this org's implicit
+        # collection.
+        PrivateDataImplicitCollection:
+            # MaxPeerCount is the maximum number of other peers (across authorized organizations)
+            # that each endorsing peer will attempt to distribute the private data to. If an
+            # endorsing peer becomes unavailable between endorsement time and commit time, other
+            # peers that are collection members but who did not yet receive the private data at
+            # endorsement time, will be able to pull the private data from peers the private data
+            # was disseminated to. If this value is set to 0, the private data is not disseminated
+            # at endorsement time, forcing private data pulls against endorsing peers on all authorized
+            # peers at commit time.
+            MaxPeerCount: 1
```

Comment:
why is this 1 by default? Shouldn't it be high like 100 or something?

Reply:
Wasn't sure how relevant the sample config was supposed to be. `1` was never supposed to be considered as a default. As a matter of fact currently the property doesn't even need to be specified and will just default to 0 if not specified. Of course this can all be changed to support actual defaults and suggested sample values if necessary. I'm open to suggestions.

Reply:
We don't need to change the code, we should just put here a sane default which is in my opinion, to send to everyone that is eligible.

Reply:
Sounds good. I'll talk to @denyeart to see if he has a preference for what it should be.

Reply:
Talked to Dave and we decided on both a code level default for when the properties aren't specified as well as sampleconfig default of `requiredPeerCount: 0` and `maxPeerCount: 1`.